docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -132,5 +131,4 @@ Hello World!
 ----
 
 [NOTE]
-At this point data is still stored on the previously existing nodes. For more information about migrating the actual data from the old
-to the new deployment, see xref:migrating-object-storage-data-to-rhoso-nodes_migrate-object-storage-service[Migrating the {object_storage_first_ref} data from {OpenStackShort} to {rhos_long} nodes].
+The Object Storage data is still stored on the existing {OpenStackShort} nodes. For more information about migrating the actual data from the {OpenStackShort} deployment to the {rhos_long} deployment, see xref:migrating-object-storage-data-to-rhoso-nodes_migrate-object-storage-service[Migrating the {object_storage_first_ref} data from {OpenStackShort} to {rhos_long} nodes].
```

Comment:
@cschwede Please confirm that my edits to this note are accurate, particularly the first sentence.

Reply:
LGTM.